src/components/cards/TradeCard/TradeCard.vue

```diff
@@ -48,6 +48,12 @@
         block
         @click.prevent="connectWallet"
       />
+      <BalBtn
+        v-else-if="networkMismatch"
+        :label="$t('networkMismatch', [appNetwork.name])"
+        block
+        @click.prevent="changeNetwork"
+      />
```

Comment:
It probably makes sense to use `BalAlert` here the same way we use it to show other errors (#387).

Reply:
I'm not sure that this would be a great UX change. 

I feel that the "Trade" button should clearly signal what users need to do next and make it as easy as possible.
By using `BalAlert`, we're moving the main action we're looking to push users towards off of the main focal point of the trade UI.

By the same token, I don't think we should have removed the "Connect Wallet" behaviour from that button.